### ISSUE TITLE
Support 24h clock

### ISSUE THREAD

**Issue by galan** [Ulauncher/ulauncher-timer#12]
Hi, when trying to use the timer using eg. `ti 18:50 Hello` it states that it is an incorrect request. Only am/pm seems to be supported.

**Comment by vanrein** [comment]
Yes, am/pm are really local choices, and it should be trivial to add syntax without AM or PM added.  It would even offer a shorthand that allows dropping AM.

**Comment by brunetton** [comment]
Fixed in #20
